front-end/src/screens/MenuBar/MobileMenu.tsx

```diff
@@ -0,0 +1,221 @@
+import React, { ReactNode, useContext, useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { Container, Menu, Icon, Sidebar, Sticky } from 'semantic-ui-react';
+import styled from '@xstyled/styled-components';
+
+import { UserDetailsContext } from '../../context/UserDetailsContext';
+import { useLogoutMutation } from '../../generated/auth-graphql';
+import { useRouter } from '../../hooks';
+import { logout } from '../../services/auth.service';
+
+interface Props {
+	children?: ReactNode,
+	className?: 'mobilemenu'
```

Comment:
This is a type definition interface, so you shouldn't define the className here.
```suggestion
	className?: string
```